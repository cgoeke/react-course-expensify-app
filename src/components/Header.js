import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink to="/" activeClassName="" exact={true}>Dashboard</NavLink>
        <NavLink to="/create" activeClassName="">Create Expense</NavLink>
        <NavLink to="/help" activeClassName="">Help</NavLink>
    </header>
); 

export default Header;