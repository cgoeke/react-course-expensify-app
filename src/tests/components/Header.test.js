import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('renders Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});