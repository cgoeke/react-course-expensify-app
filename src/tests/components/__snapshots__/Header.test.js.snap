// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName=""
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName=""
    aria-current="page"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName=""
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
